lib/simulator.js

```diff
@@ -771,11 +771,13 @@ function findSimulators(options, callback) {
 				}
 
 				if (!simHandle) {
+					const helpText = '\n\nPlease open Xcode, navigate to "Window > Devices and Simulators" and create a new Simulator with your preferred configuration.';
```

Comment:
If no Simulator is available, this can also mean that a Simulator was deleted or corrupted (even in earlier Xcode versions). That way, we can show this message for all cases where a Simular does not exist.

Reply:
Ah, good point!